content/FLASHTnT/FLASHTnTLayoutManager.py

```
@@ -18,12 +22,37 @@
+    # Check if layout has been set
+    if not file_manager.result_exists('layout', 'layout'):
+        return None
+    # fetch layout from cache
+    layout = file_manager.get_results('layout', 'layout')['layout']
+
+    return layout['layout'], layout['side_by_side'] 
 
 def resetSettingsToDefault(num_of_exp=1):
     st.session_state["layout_setting_tagger"] = [[['']]] # 1D: experiment, 2D: row, 3D: column, element=component name
     st.session_state["num_of_experiment_to_show_tagger"] = num_of_exp
     for index in range(1, num_of_exp):
```

Ruff (0.8.2)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)